api/models/record.js

```diff
@@ -58,6 +58,36 @@ const Record = BaseModel.extend({
       return this.related('trial').url;
     }
   },
+  trialsPerRegistry: function () {
+  return bookshelf.knex
+    .select(
+      bookshelf.knex.raw('primary_register as registry'),
+      bookshelf.knex.raw('count(primary_register)::int')
+    )
+    .from('trialrecords')
+    .groupBy('primary_register')
+    .orderBy('primary_register')
+    .then((rows) => {
+      return rows;
+    });
+  },
+  lastRegistryUpdate: function () {
```

Comment:
From what I understood, `lastRegistryUpdate` is the latest time we've updated data from a registry, right? If that's the case, it feels to me that it should be in the `Source` model, and not here. A sign that it doesn't belong in `Record` is that all attributes are related to the `Source`.